src/Render/Markdown/DocumentContext.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace D2L.Dev.Docs.Render.Markdown {
+	// TODO: Add more properties for context
+	// Some possibilities could be github repo url, list of docs repos
+	internal sealed class DocumentContext {

```

Comment:
This is meant to be a way for us to provide metadata about the document being rendered to our custom renderers. 
Currently it just stores the name of the repository the doc lives in, but as we need things they can be added here.